XVim2/XVim/XVim.m

```diff
@@ -166,7 +166,9 @@ - (id)init
                                                        object:nil];
         }
         else {
-            [self addMenuItem];
+            dispatch_async(dispatch_get_main_queue(), ^{
+                [self addMenuItem];
+            });
```

Comment:
This is fix when load via MacForge.

```
Process:               Xcode [6791]
Path:                  /Applications/Xcode.app/Contents/MacOS/Xcode
Identifier:            Xcode
Version:               12.4 (17801)
Build Info:            IDEFrameworks-17801000000000000~7 (12D4e)
Code Type:             X86-64 (Native)
Parent Process:        ??? [1]
Responsible:           Xcode [6791]
User ID:               501

Date/Time:             2021-04-04 00:15:41.823 +0900
OS Version:            macOS 11.2.1 (20D74)
Report Version:        12
Bridge OS Version:     5.2 (18P4346)
Anonymous UUID:        806AEC77-75B7-48EE-8368-FE248094F0D8


Time Awake Since Boot: 2300 seconds

System Integrity Protection: disabled

Crashed Thread:        0  Dispatch queue: com.apple.main-thread

Exception Type:        EXC_BAD_INSTRUCTION (SIGILL)
Exception Codes:       0x0000000000000001, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Termination Signal:    Illegal instruction: 4
Termination Reason:    Namespace SIGNAL, Code 0x4
Terminating Process:   exc handler [6791]

External Modification Warnings:
Thread creation by external task.

Application Specific Information:
ProductBuildVersion: 12D4e
BUG IN CLIENT OF LIBDISPATCH: trying to lock recursively

Thread 0 Crashed:: Dispatch queue: com.apple.main-thread
0   libdispatch.dylib             	0x00007fff2048bff2 _dispatch_once_wait.cold.1 + 14
1   libdispatch.dylib             	0x00007fff2045ef3b _dispatch_once_wait + 93
2   net.JugglerShu.XVim2          	0x000000013f367b90 _dispatch_once + 19 (once.h:84) [inlined]
3   net.JugglerShu.XVim2          	0x000000013f367b90 +[XVim instance] + 64 (XVim.m:124)
4   net.JugglerShu.XVim2          	0x000000013f368cfd -[XVim xvimMenuItem] + 253 (XVim.m:315)
5   net.JugglerShu.XVim2          	0x000000013f368b35 -[XVim addMenuItem] + 133 (XVim.m:295)
6   net.JugglerShu.XVim2          	0x000000013f367fa4 -[XVim init] + 932 (XVim.m:169)
7   net.JugglerShu.XVim2          	0x000000013f367bc5 __16+[XVim instance]_block_invoke + 37 (XVim.m:125)
8   libdispatch.dylib             	0x00007fff2045e7c7 _dispatch_client_callout + 8
9   libdispatch.dylib             	0x00007fff2045f96b _dispatch_once_callout + 20
10  net.JugglerShu.XVim2          	0x000000013f367b90 _dispatch_once + 19 (once.h:84) [inlined]
11  net.JugglerShu.XVim2          	0x000000013f367b90 +[XVim instance] + 64 (XVim.m:124)
12  net.JugglerShu.XVim2          	0x000000013f35cdfd -[XVimWindow handleRelativeNumbers] + 61 (XVimWindow.m:519)
13  net.JugglerShu.XVim2          	0x000000013f35ac34 -[XVimWindow resetEvaluatorStack:activateNormalHandler:] + 148 (XVimWindow.m:116)
14  net.JugglerShu.XVim2          	0x000000013f35a67e -[XVimWindow initWithSourceView:] + 270 (XVimWindow.m:60)
15  net.JugglerShu.XVim2          	0x000000013f38685e -[XVimIDESourceEditorView xvim_window] + 238 (XVimIDESourceEditorView.m:97)
16  net.JugglerShu.XVim2          	0x000000013f386748 -[XVimIDESourceEditorView xvim_setupOnFirstAppearance] + 24 (XVimIDESourceEditorView.m:88)
17  com.apple.Foundation          	0x00007fff2146f01c __NSBLOCKOPERATION_IS_CALLING_OUT_TO_A_BLOCK__ + 7
18  com.apple.Foundation          	0x00007fff2146ef14 -[NSBlockOperation main] + 98
19  com.apple.Foundation          	0x00007fff2146eea5 __NSOPERATION_IS_INVOKING_MAIN__ + 17
20  com.apple.Foundation          	0x00007fff2146e156 -[NSOperation start] + 785
21  com.apple.Foundation          	0x00007fff2146de3a __NSOPERATIONQUEUE_IS_STARTING_AN_OPERATION__ + 17
22  com.apple.Foundation          	0x00007fff2146dd0d __NSOQSchedule_f + 182
23  libdispatch.dylib             	0x00007fff2046b1d5 _dispatch_block_async_invoke2 + 83
24  libdispatch.dylib             	0x00007fff2045e7c7 _dispatch_client_callout + 8
25  libdispatch.dylib             	0x00007fff2046ab86 _dispatch_main_queue_callback_4CF + 940
26  com.apple.CoreFoundation      	0x00007fff2073e970 __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ + 9
27  com.apple.CoreFoundation      	0x00007fff20700852 __CFRunLoopRun + 2731
28  com.apple.CoreFoundation      	0x00007fff206ff6ce CFRunLoopRunSpecific + 563
29  com.apple.HIToolbox           	0x00007fff28987630 RunCurrentEventLoopInMode + 292
30  com.apple.HIToolbox           	0x00007fff2898742c ReceiveNextEventCommon + 709
31  com.apple.HIToolbox           	0x00007fff2898714f _BlockUntilNextEventMatchingListInModeWithFilter + 64
32  com.apple.AppKit              	0x00007fff22f1f9b1 _DPSNextEvent + 883
33  com.apple.AppKit              	0x00007fff22f1e177 -[NSApplication(NSEvent) _nextEventMatchingEventMask:untilDate:inMode:dequeue:] + 1366
34  com.apple.dt.DVTKit           	0x0000000101eabd07 -[DVTApplication nextEventMatchingMask:untilDate:inMode:dequeue:] + 421
35  com.apple.AppKit              	0x00007fff22f1068a -[NSApplication run] + 586
36  com.apple.AppKit              	0x00007fff22ee496f NSApplicationMain + 816
37  libdyld.dylib                 	0x00007fff20624621 start + 1
```